.history/.github/actions/hoge/action_20241004015255.yml

```
@@ -0,0 +1,42 @@
+            "https://api.github.com/repos/${{ github.repository }}/pulls/$PR_NUMBER" | jq -r '.body')
+          
+          # Use the provided keyword or default to 'default-keyword'
+          KEYWORD="${{ inputs.keyword }}"
+          START_TAG="<!--${KEYWORD}:start-->"
+          END_TAG="<!--${KEYWORD}:end-->"
+          
+          # Check if the text is already present between the tags
+          if ! echo "$PR_DESCRIPTION" | grep -q "${{ inputs.text }}"; then
+            # Insert text between the specified comment tags
+            UPDATED_DESCRIPTION=$(echo "$PR_DESCRIPTION" | sed "/$START_TAG/,/$END_TAG/{/$START_TAG/{n;s/.*/${{ inputs.text }}/;}}")
+            
```

yamllint

[error] trailing spaces

(trailing-spaces)